resources/prometheus/prometheus-rules.yaml

```diff
@@ -291,6 +291,40 @@ spec:
             description: |
               Remaining DB instances: {{ with query "acs_fleetshard_central_db_instances_max - acs_fleetshard_central_db_instances_used" }}{{ . | first | value | humanize }}{{ end }}. A quota increase must be requested from AWS.
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-029-increase-aws-rds-limits.md"
+
+    - name: aws-ses
+      rules:
+        - alert: AWSSESReputationBounceRateTooHigh
+          expr: avg_over_time(aws_ses_reputation_bounce_rate_sum[1h]) > 0.05
+          for: 1s
+          labels:
+            severity: warning
+          annotations:
+            summary: 'AWS SES Bounce rate too high'
+            description: 'The SES bounce rate is {{ $value | humanizePercentage }}, which is higher than the maximum limit of 5%.'
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-040-aws-ses-violation.md"
+        - alert: AWSSESReputationComplaintRateTooHigh
+          expr: avg_over_time(aws_ses_reputation_complaint_rate_sum[1h]) > 0.01
+          for: 1s
+          labels:
+            severity: warning
+          annotations:
+            summary: 'AWS SES Complaint rate too high'
+            description: 'The SES complaint rate is {{ $value | humanizePercentage }}, which is higher than the maximum limit of 1%.'
+            sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-040-aws-ses-violation.md"
+        - alert: AWSSESSendRateTooHigh
+          # cloudwatch is setup to collect aws_ses_send_sum in 2 minute intervals (120 seconds)
+          # and the max send rate is 14/s.
+          expr: (max_over_time(aws_ses_send_sum[1h]) / 120) > 12
+          for: 1s
+          labels:
+              severity: warning
+          annotations:
+              summary: 'AWS SES Send rate too high'
+              description: 'The SES send rate is {{ $value }} per second, which is dangerously approaching the maximum limit of 14 per second.'
```

Comment:
The description is a bit misleading because the value is the max over the last hour, but it reads like the current send rate value.